Makefile

```diff
@@ -34,6 +34,24 @@ env: kafl/env.sh
 	@echo "Entering environment in sub-shell. Exit with 'Ctrl-d'."
 	@PROMPT_COMMAND='source kafl/env.sh; unset PROMPT_COMMAND' $(SHELL)
 
+define HELP_TEXT
+Manage kAFL installation.
+
+  Supported actions:
+    deploy:\tUpdate components and perform full installation
```

Comment:
i would just lay it as `perform full installation`.
updating the components is specific behavior that can be toggled with tags, so it shouldn't come up here as the a general description of the functionality of `deploy`

Reply:
I think the `deploy` action will also perform a git update, or? This may be surprising for users who just want to do a reinstall, so I added it here. But I agree that is a topic to be resolved elsewhere potentially..

Reply:
kind of done